src/scenes/game-scene.ts

```diff
@@ -109,17 +109,22 @@ const soundConfig = [
     filePath: './audio/music/spook.mp3',
     loop: true,
   },
+  {
+    key: 'jump',
+    filePath: './audio/sfx/jump.mp3',
+    loop: false,
+  },
```

Comment:
Nice 👍 I'm wondering if we have options to toggle soundFx/music on/off now 🤔

Reply:
like separate controls for sfx vs music?
yeah its defos something ive seen and used before
like if i want to play a game, but am sick of the music they have, i can just get sfx from the game and play my own music
i like it

Reply:
lets make a card in trello for it
would be a good for next jam if we find we dont have time to do this idea